### PR TITLE
Bridge test to work with sha

### PR DESCRIPTION
**Pull request by girazoki**
UPdates download job so and adds more debuggeability to e2e bridge tests

### CODE REVIEW

File: .github/workflows/e2e-test-bridge.yml [hunk 1]
```diff
@@ -93,13 +93,18 @@ jobs:
...
-              uses: dawidd6/action-download-artifact@v2
+              uses: dawidd6/action-download-artifact@v7
               with:
-                run_id: ${{ steps.retrieve-run-id.outputs.run_id }}
+                commit: ${{ steps.retrieve-run-id.outputs.run_sha }}
+                workflow: release.yml
                 name: binaries
                 path: target/release
-                search_artifacts: true
+                allow_forks: true
```
Comment (ParthDesai) [inline .github/workflows/e2e-test-bridge.yml:107]: Why do you need to have `allow_forks: true`?

Comment (girazoki) [reply .github/workflows/e2e-test-bridge.yml:107]: I read the guy recommends it

Comment (girazoki) [reply .github/workflows/e2e-test-bridge.yml:107]: besides we were running with allow_forks already, it's just in the new version the default is false

Comment (ParthDesai) [reply .github/workflows/e2e-test-bridge.yml:107]: If we do not need it, let's set it to false as making it true explicitly allows this attack: https://github.com/dawidd6/action-download-artifact/security/advisories/GHSA-5xr6-xhww-33m4

Which is the reason why it is by default false.

Comment (girazoki) [reply .github/workflows/e2e-test-bridge.yml]: fixed

**Review by NindoK — APPROVED** [review]

**Pull request merged**